test/get-changelog-test.js

```diff
@@ -243,6 +246,68 @@ describe(getChangelog, function() {
     expect(changelog).to.include('* old-release');
   });
 
+  it('can generate less than one release', async function() {
+    fixturify.writeSync(tmpPath, {
+      'packages': {
+        'my-app': {
+          'package.json': stringifyJson({
+            'name': '@scope/my-app',
+            'version': '1.0.0',
+          }),
+        },
+      },
+      'package.json': stringifyJson({
+        'private': true,
+        'workspaces': [
+          'packages/*',
+        ],
+      }),
+    });
+
+    await execa('git', ['add', '.'], { cwd: tmpPath });
+    await execa('git', ['commit', '-m', 'chore: release'], { cwd: tmpPath });
+
+    process.chdir(path.join(tmpPath, 'packages/my-app'));
+
+    await standardVersion({
+      path: path.join(tmpPath, 'packages/my-app'),
+      tagPrefix: '@scope/my-app@',
+      firstRelease: true,
+    });
+
+    fixturify.writeSync(tmpPath, {
+      'packages': {
+        'my-app': {
+          'index.js': 'foo',
+        },
+      },
+    });
+
+    await execa('git', ['add', '.'], { cwd: tmpPath });
+    await execa('git', ['commit', '-m', 'fix: ignored change'], { cwd: tmpPath });
+
+    let commit = await getCurrentCommit(tmpPath);
+
+    fixturify.writeSync(tmpPath, {
+      'packages': {
+        'my-app': {
+          'index.js': 'bar',
+        },
+      },
+    });
+
+    await execa('git', ['add', '.'], { cwd: tmpPath });
+    await execa('git', ['commit', '-m', 'fix: included change'], { cwd: tmpPath });
+
+    let changelog = await getChangelog({
```

Comment:
so this is `(exclusive, inclusive]`? `(ignored change, included change]`

Reply:
correct

Reply:
`(not going in changelog, going in changelog]`

Reply:
should `fromCommit` be changed to `afterCommit` then?

from, to me, implies that it'll be included. It wasn't until reading the last two assertions at the bottom of this test that I knew that they `fromCommit` was excluded

Reply:
Your not wrong, but this is a convention already. I pass it into the changelog generator's `from` option. Also, `fromCommit` is an option of `changed` and `changedFiles` functions. We may have to rethink this as a whole to see if it's worth changing.